inginious-problems-blockly/templates/box_blockly.html

```diff
@@ -101,8 +101,8 @@ <h4 class="modal-title">{{blockly_dico["name"]}}</h4>
 <script src="{{get_homepath()}}/plugins/blockly/static/blockly/python_compressed.js" type="text/javascript" charset="utf-8"></script>
 <script src="{{get_homepath()}}/plugins/blockly/static/blockly/javascript_compressed.js" type="text/javascript" charset="utf-8"></script>
 
-<!-- Blockly English Translation -->
-<script src="{{get_homepath()}}/plugins/blockly/static/blockly/msg/js/en.js" type="text/javascript" charset="utf-8"></script>
+<!-- Blockly Language Translation -->
+<script src="{{get_homepath()}}/plugins/blockly/static/blockly/msg/js/{{user_manager.session_language()}}.js" type="text/javascript" charset="utf-8"></script>
```

Comment:
There is almost no chance such an issue happens, but you should check if the file actually exists before including it, and insert the English one as default.